packages/checkup-plugin-ember/__tests__/ember-addon-engine-count-task-test.ts

```diff
@@ -0,0 +1,56 @@
+import { stdout, getTaskContext, EmberProject } from '@checkup/test-helpers';
+import { getPluginName } from '@checkup/core';
+import EmberAddonEngineCountTask from '../src/tasks/ember-addon-engine-count-task';
+import EmberAddonEngineCountTaskResult from '../src/results/ember-addon-engine-count-task-result';
+
+describe('ember-addon-engine-count-task', () => {
+  let emberProject: EmberProject;
+  let pluginName = getPluginName(__dirname);
+
+  beforeEach(() => {
+    emberProject = new EmberProject('checkup-app', '0.0.0', (emberProject) => {
+      emberProject.addDependency('ember-cli', '^3.15.0');
+    });
+    emberProject.addInRepoAddon('ember-cli-blueprint-test-helpers', 'latest');
+    emberProject.addInRepoAddon('ember-cli-shmooprint-test-helpers', 'latest');
```

Comment:
I don't think we'd have in-repo addons named like this. Can we just add something that seems more realistic?

```suggestion
    emberProject.addInRepoAddon('admin', 'latest');
    emberProject.addInRepoAddon('shopping-cart', 'latest');
```